lsls/lsls.c

```diff
@@ -1,18 +1,70 @@
 #include <stdio.h>
 #include <dirent.h>
+#include <stdlib.h>
+#include <sys/stat.h>
 
 /**
  * Main
  */
 int main(int argc, char **argv)
 {
   // Parse command line
+  char *directory_name;
+  if (argc == 1)
+  {
+    directory_name = ".";
+  }
+  else if (argc == 2)
+  {
+    directory_name = argv[1];
+  }
+  else
+  {
+    fprintf(stderr, "Too amny arguments! Usage: ./index2 [your directory name]\n");
+    exit(1);
+  }
 
```

Comment:
consider commenting your code...but otherwise good, clean and readable...nice job!